src/redux/saga/employer.ts

```diff
@@ -0,0 +1,19 @@
+import { put, all, takeLatest, select } from 'redux-saga/effects';
+import axios from 'axios';
+
+import { registerEmployerSuccess } from '../../redux/actions/employer';
+
+function* registerEmployerSaga(credentials: Credential) {
+  try {
+    const req = yield axios.post('/employer', { credentials });
+    yield put(registerEmployerSuccess(req.data));
+  } catch (error) {
+    console.log(error);
+  }
+}
```

Comment:
If I hit the https://tindev-dev-deploy.herokuapp.com/employer here I'll get the cors error, so maybe we should use cors in the backend?

Reply:
Can you create an issue on the backend for this ?